### ISSUE TITLE
Add additional advanced search filters for new facility fields from #800

### ISSUE THREAD

**Issue by vegu** [peeringdb/peeringdb#1016]
This was originally part of #802 but was moved out and put on hold after #800 had to be moved back into `Decide` for some changes and further review.

#800 is ready to be implemented and worked on now.

**Summary**

Add following fields to advanced search:
- facility: Unique and Diverse Serving Substations
- facility: Property Owner / Lessee
- facility: Available Voltage Service
